test/api.test.ts

```diff
@@ -69,6 +70,19 @@ describe("api", () => {
         await executeRemoveDependencySpy();
         expect(removeDependencySpy).toHaveReturned();
       }, 60_000);
+
+      it("runs script", async () => {
+        const runScriptSpy = vi.fn(runScript);
+
+        const executeRunScriptSpy = () =>
+          runScriptSpy("test-script", {
+            cwd: fixture.dir,
+            silent: !process.env.DEBUG,
+          });
+
+        await executeRunScriptSpy();
```

Comment:
Not sure to understand how this test works.. we might need a better strategy. for example a script to touch a file so we know it ran by package manager